meson.build

```diff
@@ -150,8 +150,25 @@ lib_wlr = library(
 	install: true,
 	link_args: symbols_flag,
 	link_depends: symbols_file,
+	prelink: true,
 )
 
+if get_option('default_library') != 'shared'
+	lib_target = lib_wlr
+	if get_option('default_library') == 'both'
+		lib_target = lib_wlr.get_static_lib()
+	endif
+	objcopy_prog = find_program('objcopy', native: true)
+	custom_target('libwlroots',
+		input: lib_wlr,
+		output: lib_wlr.name() + '.is-stripped',
+		command: [
+			'sh', '-c', 'objcopy -w -L "!wlr*" -L "!_wlr*" -L "*" @INPUT@; touch @OUTPUT@'.format(objcopy_prog.full_path())
```

Comment:
Instead of hardcoding `objcopy`, we probably want to use `@0@`

Reply:
Yeah, this format string was very broken. Side-stepped the shell entirely now.